packages/fxa-payments-server/src/components/PaymentProviderDetails/index.scss

```diff
@@ -46,7 +46,7 @@
   background-image: url('../../images/paypal.svg');
 }
 
-.unbranded::before {
+.unknown::before {
```

Comment:
I removed the "unbranded" class in PaymentProviderDetails' stylesheet, because we were never applying that class. In the case that `brand` is `null`, we don’t even render the `PaymentProviderDetails` component.

Interestingly, while `customer.brand` can be `null` from the auth server, [`charge?.payment_method_details?.card?.brand`](https://stripe.com/docs/api/charges/object#charge_object-payment_method_details-card-brand) can never be `null` from Stripe, so I don't think this is actually a scenario we will ever see. Stripe provides a value of "unknown" for a card brand that doesn't match any of the other known supported brands.